File: Modul 4/Soal 5/PRAK405-2410817210014-ZAVIERGHANI.c
#include <stdio.h>

int main() {
    int n, kelipatan;
    int total = 0;

    printf("Masukkan baris pertama (n) dan kelipatan (dipisahkan dengan spasi): ");
    scanf("%d %d", &n, &kelipatan);

    for (int i = 1; i <= n; i++) {
        int sum = 0;

        for (int j = i; j >= 1; j--) {
            sum += j * kelipatan;
            if (j > 1) {
                printf("(%d * %d) + ", j, kelipatan);
            } else {
                printf("(%d * %d) = %d\n", j, kelipatan, sum);
            }
        }

        total += sum;  
    }

    printf("%d\n", total);

    return 0;
}

File: Modul 4/Soal 5/PRAK405-2410817210014-ZAVIER GHANI.py
def hitung_kelipatan():
    n, kelipatan = map(int, input("Masukkan baris pertama (n) dan kelipatan (dipisahkan dengan spasi): ").split())
    
    total = 0
    
    for i in range(1, n + 1):
        sum_kelipatan = 0
        for j in range(i, 0, -1):
            sum_kelipatan += j * kelipatan
            if j > 1:
                print(f"({j} * {kelipatan}) + ", end="")
            else:
                print(f"({j} * {kelipatan}) = {sum_kelipatan}")
        
        total += sum_kelipatan 
    print(total)

hitung_kelipatan()
